packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -569,7 +569,11 @@ export default connectStore(() => {
             <tr>
               <th>{_('xenToolsStatus')}</th>
               <td>
-                {vm.xenTools && `${vm.xenTools.major}.${vm.xenTools.minor}`}
+                {vm.xenTools &&
+                vm.xenTools.major !== undefined &&
+                vm.xenTools.minor !== undefined
```

Comment:
This shouldn't be necessary because it should be handled in xo-server. Maybe this test should be changed:
https://github.com/vatesfr/xen-orchestra/blob/945a2006c967e721537cbde74c006e50fde04c7b/packages/xo-server/src/xapi-object-to-xo.js#L230-L232